packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -240,15 +262,26 @@ class ScrollableBase extends Component {
 		if (pos) {
 			const
 				{top, left} = pos,
-				bounds = this.uiRef.getScrollBounds();
+				bounds = this.uiRef.getScrollBounds(),
+				horizontalMove = bounds.maxLeft > 0 && left !== this.uiRef.scrollLeft,
+				verticalMove = bounds.maxTop > 0 && top !== this.uiRef.scrollTop;
 
-			if ((bounds.maxTop > 0 && top !== this.uiRef.scrollTop) || (bounds.maxLeft > 0 && left !== this.uiRef.scrollLeft)) {
+			if (horizontalMove || verticalMove) {
 				this.uiRef.start({
 					targetX: left,
 					targetY: top,
 					animate: (animationDuration > 0) && this.animateOnFocus,
 					duration: animationDuration
 				});
+
+				if (this.childRef.shouldPreventOverscrollEffect ? !this.childRef.shouldPreventOverscrollEffect() : true) {
```

Comment:
It would be better to use the following condition instead of `?`.
```
!this.childRef.shouldPreventOverscrollEffect || !this.childRef.shouldPreventOverscrollEffect()
```

Reply:
It is shorter than the current code, but it looks little unclear.

Reply:
Oh, it was the condition inside 'if' statement. I will fix it.